freqtrade/optimize/backtesting.py

```diff
@@ -497,7 +501,7 @@ def _get_sell_trade_entry(self, trade: LocalTrade, sell_row: Tuple) -> Optional[
         sell_candle_time: datetime = sell_row[DATE_IDX].to_pydatetime()
 
         if self.trading_mode == TradingMode.FUTURES:
-            # TODO-lev: Other fees / liquidation price?
+            # TODO-lev: liquidation price?
```

Comment:
Futures has funding_fees + regular trading fees that you would see in spot, unless there's a liquidation. Are we going to set our liq price to 5% above the actual liq price?

Reply:
I'm not certain i understand this (and if you mean this only for backtesting, or for live mode too).

i mean  - would we assume that liquidation fees are 5%, and therefore have backtesting "pay that as a fee" - or would we force the stop 5% above liquidation price to ensure we're not running into a real liquidation? (again - only for backtesting, or also for live??)

Reply:
No I'm saying we never have an actual liquidation, and set our liquidation price as the actual liquidation price + 5%

Reply:
yeah, something like "liquidation_offset" as parameter might work